ulib/experimental/Steel.Semantics.Hoare.MST.fst

```diff
@@ -1428,11 +1431,11 @@ let rec run
   | Ret _ x _ -> x
 
   | _ ->
-    let m0 = get () in
+    let m0 : full_mem st = get () in
     let Step new_pre new_post _ _ f = step f in
-    let m1 = get () in
+    let m1 : full_mem st = get () in
     let x = run f in
-    let m2 = get () in
+    let m2 : full_mem st = get () in
```

Comment:
hi @aseemr. See many diffs like this where I have to annotate the result of get as `full_mem st` otherwise the proof fails. Wonder why.